### PR TITLE
Raise the resource limit on open files for fedmsg-hub

### PR DESCRIPTION
**Pull request by jeremycline**
This is a short-term solution to systems running into the maximum open
file descriptor limit.

Signed-off-by: Jeremy Cline <jeremy@jcline.org>

### CODE REVIEW

**Comment by ralphbean** [comment]
:+1: from me!

File: fedmsg/commands/hub.py [hunk 1]
```diff
@@ -17,11 +17,20 @@
 #
 # Authors:  Ralph Bean <rbean@redhat.com>
 #
-import fedmsg
+from gettext import gettext as _
```
Comment (pypingou) [inline fedmsg/commands/hub.py]: do we already depend on gettext?

Comment (jeremycline) [inline fedmsg/commands/hub.py]: [gettext](https://docs.python.org/2/library/gettext.html) is in the standard library. I don't know exactly when it was introduced, but it's definitely 2.6+ so we're good.

**Comment by pypingou** [comment]
Cool :+1: for me as well then :)

**Pull request merged**